pkg/common/constants.go

```diff
@@ -57,7 +57,8 @@ const (
 )
 
 // DefaultTargetFilePattern configurations related to the write-back functionality
-const DefaultTargetFilePattern = ".argocd-source-%s.yaml"
+const DefaultTargetFilePattern = ".argocd-source-%s_%s.yaml"
```

Comment:
Would it be easier to work with if we use the same separator "-", instead of "_"?

Reply:
No : ArgoCD would not resync the application if we do not follow this pattern

Reply:
This seems to be a majorly breaking change

Reply:
I'm not sure to understand why it is a breaking change?

Reply:
Sorry, I have seen for the condition to decide which file name to actually use/write to later on.

Reply:
@sribiere-jellysmack do you have a link to the argocd code or docs that describes the default target file pattern with namespace?  All I can find is https://argo-cd.readthedocs.io/en/latest/user-guide/parameters/#store-overrides-in-git but this does not mention the file name pattern that contains namespace.